pkg/apis/resource/v1alpha1/pipelineresource_validation_test.go

```diff
@@ -136,32 +122,6 @@ func TestResourceValidation_Invalid(t *testing.T) {
 	}
 }
 
-func TestResourceValidation_Valid(t *testing.T) {
```

Comment:
If we remove this test, there's no test cases to ensure the behavior of Valid ResourceValidation... Is the plan to remove all kinds of pipelineresources (guess so?), what's the estimated timeline?

Without adding unit tests to the PipelineResource validation, we may end up in an unreleasable state :(

Reply:
This test case has been gradually removed along with the removal of each pipelineResources eg. [cloudEvent resources](https://github.com/tektoncd/pipeline/pull/5995) and [clusterResources](https://github.com/tektoncd/pipeline/pull/5996). And we are targeting to remove all pipelineResources within the next release so I think this tests now serves not the exact purpose as it did previously.

Reply:
Plus I think we might have already not been updating v1alpha1 as they were not supported, it might be safe to assume that we can delete this?

Reply:
SG, I'm okay If we are targeting at deleting everything within this release, we should update https://github.com/tektoncd/pipeline/issues/5967 to reflect that and add that issue to release v0.45.0

Reply:
Thanks @XinruZhang , updated at https://github.com/tektoncd/pipeline/issues/5967#issuecomment-1405011306